dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -439,6 +439,7 @@ function handleServerside(
     const fetchCallback = () => {
         const headers = getCSRFHeader() as any;
         let url = `${urlBase(config)}_dash-update-component`;
+        let new_body = body;
```

Comment:
We use camelCase for variable that are strictly js based, usually if there is a snake_case variable in the js code it's because it's transfered back and forth with the backend or part of the clientside api.